Feature mission rework

ThomasMattern2 opened the pull request:
👑🏀🐐 👑🏀🐐 👑🏀🐐 👑🏀🐐 👑🏀🐐 👑🏀🐐 👑🏀🐐

ThomasMattern2 committed:
Pull up mission refactor

ThomasMattern2 committed:
Pull up speed changes

ThomasMattern2 committed:
Pull in minor fixes to missions also added drop distance to json

Lmah0 commented on server.py:
```diff
@@ -111,7 +111,7 @@ def payload_drop_mission():
         mission.upload_payload_drop_mission(vehicle_connection, payload_object_coord)
```
We'll need to update the `drop_altitude` in this `payload_object_coord` since I don't think they're comfortable flying at 18. I'll ask mech today and get back to you.

Lmah0 replied:
they are happy with this, we won't need to change this value.

Lmah0 committed:
Updating the autopilotdevelopment submodule to 'comp' branch which has the raspberry pi import statements working for it.

Also removing the planeObject and copterObject since this is breaking on the Pi.

ThomasMattern2 committed:
Update autopilot submodule

ThomasMattern2 committed:
Pull in submodule

Lmah0 committed:
Merge branch 'master' into feature-missionRework
